hw_openqa.ipynb

```diff
@@ -160,14 +147,11 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "lm = dsp.GPT3(model='text-davinci-001', api_key=openai_key)\n",
-    "\n",
-    "# Options for Cohere: command-medium-nightly, command-xlarge-nightly\n",
-    "#lm = dsp.Cohere(model='command-xlarge-nightly', api_key=cohere_key)\n",
+    "lm = dspy.OpenAI(model='gpt-3.5-turbo', api_key=openai_key)\n",
```

Comment:
I chose `gpt-3.5-turbo` because I thought it was the default. Let's keep it, since it usefully makes the point about how demonstrations can change predictions, and it doesn't seem productive to keep updating things as the `instruct` model is updated.